src/components/App.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Routes, BrowserRouter, Route } from 'react-router-dom';
+import AddRoom from './Rooms/AddRoom';
+import Rooms from './Rooms/Rooms';
+import DeleteRoom from './Rooms/DeleteRoom';
+import AddUser from './user';
+import Room from './Rooms/Room';
+import Nav from './Nav';
+
+function App() {
+  return (
+    // make the design responsive only for desktop
+    <div className="row m-0 p-0">
+      <div className="col-md-2 w-sm-0 p-0">
+        <Nav />
+      </div>
+      <div className="col-md-10 p-0 vh-100">
+        <BrowserRouter>
+          <Routes>
+            <Route path="/" element={<Rooms />} />
+            {/* <Route path="/reservations" element={<MyReservations />} /> */}
+            {/* <Route path="/reservations-new" element={<ReservationForm />} /> */}
+            <Route path="/room-new" element={<AddRoom />} />
+            <Route path="/room/:id" element={<Room />} />
+            {/* <Route path="/room-list" element={<IndexRemoveRoom />} /> */}
+            <Route path="/room-delete" element={<DeleteRoom />} />
+            <Route path="/user" element={<AddUser />} />
+          </Routes>
+        </BrowserRouter>
+      </div>
+    </div>
+  );
+}
```

Comment:
- [ ] When I try to make a reservation from a room's description, I get a blank screen and no reservation is placed:

![image](https://user-images.githubusercontent.com/57421823/195332148-b701846b-fcfc-49a3-84ed-5b9abca30177.png)

- [ ] Also, when I click on "my reservations" I get a blank screen too:

![image](https://user-images.githubusercontent.com/57421823/195332313-26bc1403-1715-424a-a342-c70b740ca58e.png)

- [ ] Finally,in the "make reservations" section I am greeted again with a blank screen:

![image](https://user-images.githubusercontent.com/57421823/195332513-50109a22-8e96-413c-ace4-45e20abe6e68.png)

## Help and tips

I can see that:

1. In App.js, the reservations routes have been commented out.
2. At the time of this review, there is an PR from 6 days ago that adds the missing routes, but it is unmerged.

Kindly fix these bugs to ensure your app works as intended and users are able to make reservations. 💪

Reply:
Thank you so much for the feedback, Miguel. @felixodette  will handle this issue.